site/src/pages/WorkspaceSettingsPage/WorkspaceSchedulePage/WorkspaceScheduleForm.tsx

```diff
@@ -404,7 +399,7 @@ export const WorkspaceScheduleForm: FC<
           <TextField
             {...formHelpers("ttl", ttlShutdownAt(form.values.ttl), "ttl_ms")}
             disabled={isLoading || !form.values.autostopEnabled}
-            inputProps={{ min: 0, step: 1 }}
+            inputProps={{ min: 0, step: "any" }}
```

Comment:
@Parkreiner Good eye! Don't worry - we [still validate](https://github.com/coder/coder/pull/10707/files#diff-5fab0402c3d28232846727f8521ce0e526f79b703d404ba4496f7ceb5170f43bR173-R174) against negative and 0 inputs here.

I am using a step of 'any' because this defaults to a step of 1 while still allowing for floats.